File: bhaskara/bhaskara.py
import math

a = float(input("a: "))
b = float(input("b: "))
c = float(input("c: "))

print("\n")

delta = (b**2) - 4 * (a*c)
print(f"Delta: {delta}")
if (delta==0):
    x1 = (-b + math.sqrt(delta)) / (2 * a)
    print(f"X1: {x1}")
else:
    x1 = (-b + math.sqrt(delta)) / (2 * a)
    x2 = (-b - math.sqrt(delta)) / (2 * a)
    print(f"x1: {x1}")
    print(f"x2: {x2}")

File: bhaskara/bhaskara.c
#include <stdio.h>
#include <math.h>

int main(){
    double a,b,c;

    printf("a: \n");
    scanf("%lf", &a);
    printf("b: \n");
    scanf("%lf", &b);
    printf("c: \n");
    scanf("%lf", &c);

    double delta = (pow(b,2) - 4 * a*c);
    printf("\nDelta: %lf\n", delta);

    if(delta==0){
        float x1 = (b + sqrt(delta)) / (2*a);
        printf("x1: %lf\n", x1);
    }else{
        float x1 = (-b + sqrt(delta)) / (2*a);
        float x2 = (-b - sqrt(delta)) / (2*a);
        printf("x1: %lf\n", x1);
        printf("x2: %lf\n", x2);
    }

    return 0;
}
